src/ONYXKEYS.ts

```diff
@@ -306,7 +307,7 @@ type OnyxValues = {
     [ONYXKEYS.IS_SIDEBAR_LOADED]: boolean;
     [ONYXKEYS.SHOW_DOWNLOAD_APP_BANNER]: boolean;
     [ONYXKEYS.PERSISTED_REQUESTS]: OnyxTypes.Request[];
-    [ONYXKEYS.QUEUED_ONYX_UPDATES]: OnyxTypes.QueuedOnyxUpdates;
```

Comment:
@BartoszGrajdek Could we remove `src/types/onyx/QueuedOnyxUpdates.ts` since isn't being used anymore?

Reply:
Done! @fabioh8010